Automattic/jetpack
# VideoPress: Remove blank spaces from `Logo`

Removing blank spaces from the right side of the `Logo` component.

It had a `2px` difference in `height`, related to the wrong spaces applied in the old one.

Related with #26036 

#### Changes proposed in this Pull Request:

* Update `Logo` svg code to remove white space.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `Storybook`
* Navigate to `VideoPress/Components/Logo`
* Verify that's no white space on it.

_Before_

<img width="596" alt="image" src="https://user-images.githubusercontent.com/1663717/189190375-5e1dd0fc-1033-4e66-89b5-3871ac9a84f6.png">

_After_

<img width="596" alt="image" src="https://user-images.githubusercontent.com/1663717/189190229-2b4ed9f8-11e9-4b55-ac6b-3d76a117ebb6.png">
